maintainers/google-season-of-docs-2024.md

```diff
@@ -0,0 +1,84 @@
+# Write Nix docker and cross compilation guides (Google Season of Docs 2024 Project Proposal)
+
+## What is Nix (about us)
+
+Nix is the purely functional package manager emphasizing reproducibility and reliability.
+Our first release was in 2003, we are one of the ten largest projects on GitHub for multiple consecutive years now and have a vast variety of contributors from all over the world, totaling up to over 5000 individuals. Nix finds applications in any sector using IT, with examples reaching from finances, agriculture and research to energy providers, space exploration and high performance computing. We drastically improve our users computing experience and have a unique approach to problems.
```

Comment:
> we have been one of the ten largest projects on GitHub

As far as I'm aware we still are one the ten largest projects on GitHub.

> with more than 15 000 users

15 000 users is honestly a very low assumption, esp. if you take in metrics like the ones [here](https://github.com/NixOS/infra/tree/master/metrics/fastly) which show that the nix installer alone gets downloaded around ~50k per day and we constantly have more then 200k individual nix users hit the cache. We obviously can't take those values at face because existing users updating are mixed in to the download number and on average every nix user probably has more then one machine because companies running their dc on nix are included in those numbers.

Reply:
"have been" means it's still happening.